traveler-ios-ui/TravelerKitUI/ViewControllers/ParkingQueryFormViewController.swift

```diff
@@ -0,0 +1,169 @@
+//
+//  ParkingQueryFormViewController.swift
+//  TravelerKitUI
+//
+//  Created by Ata Namvari on 2019-10-23.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import Foundation
+import TravelerKit
+
+let stringCellIdentifier = "stringCellIdentifier"
+
+let dropOffTag = 0
+let pickUpTag = 1
```

Comment:
These are used for cell tags (which expect to be int). This is more syntactic sugar than an actual enum case. If encapsulatred into an enum it would make the code less readable.

Reply:
Even if it's syntactic sugar, I'd say enumerating this still makes it cleaner.  Say, you have:
`enum DatePickerTag: Int {
   case none = -1
   case dropOff = 0
   case pickUp = 1
`
and your datePickerTag now becomes: `private var datePickerTag: DatePickerTag = .none`

Now, all of your switch statements become more readable i.e.:
`switch (query!.airportIATA, datePickerTag) {
 case (.none, .none):
 case (.none, _):
 case (.some, _):
`

Also, why are these declared in the global scope instead of the class scope?